src/main/webapp/i18n/de/assessmentDashboard.json

```
@@ -29,7 +29,7 @@
             "assessmentsDueDate": "Einreichungsfrist der Bewertungen",
             "actions": "Aktionen",
             "notReleased": "Nicht freigegeben",
-            "readGradingInstructions": "Bewertungsanweisungen lesen",
+            "readGradingInstructions": "Bewertungsanweisungen lesen and click on confirmation button below",
```

```suggestion
            "readGradingInstructions": "Lies die Bewertungsanweisungen und klicke auf den Bestätigungsknopf darunter",
```
But also depends on the comment about the english text